airflow/cli/commands/pools.py

```diff
@@ -114,20 +121,18 @@ def pool_import(filepath, verbose):
         }
 
     """
-    if not os.path.exists(filepath):
-        raise SystemExit(f"Missing pools file {filepath}")
     pools, failed = pool_import_helper(filepath)
     if len(failed) > 0:
         raise SystemExit(f"Failed to update pool(s): {', '.join(failed)}")
-    print(f"Uploaded {len(pools)} pool(s)")
+    AirflowConsole().print(f"Uploaded {len(pools)} pool(s)")
 
 
 @pools.command("export")
-@click.argument("filepath")
+@click.argument("filepath", type=click.Path())
```

Comment:
Specifying `type` as `click.Path()` here does not change the behavior, but makes the intent clearer